internal/common/db.go

```diff
@@ -145,6 +145,11 @@ func GetClustersFromDBWhere(db *gorm.DB, eagerLoading EagerLoadingState, include
 	if err != nil {
 		return nil, err
 	}
+	// we need to fetch Hosts association to allow AfterFind hook to run
```

Comment:
What's `AfterFind`? :)

Reply:
This is grom hook which is [currently](https://github.com/openshift/assisted-service/blob/49683ef8749114c5aaac1944cb821edc6ccdea4d/internal/common/db.go#L175) used to compute number of hosts for the UI